docs/networking/ipv6.md

```diff
@@ -0,0 +1,37 @@
+# IPv6
+
+{{ kops_feature_table(kops_added_ff='1.22') }}
```

Comment:
Best not to confuse people:
```suggestion
{{ kops_feature_table(kops_added_ff='1.23') }}
```

Reply:
We did feature flag it in 1.22

Reply:
Yes, but it's a totally different approach that we don't want to document or support. Mentioning 1.22 would confuse some users in thinking that it's same as what we will release in 1.23. Am I wrong?

Reply:
I suppose it depends on what the table is supposed to mean and how people interpret "experimental". I suspect there's a good chance we'll remove the feature flag for 1.23.

Reply:
I agree that the feature flag will probably be removed in 1.23. Though, kOps docs don't have a way to say, experimental in version X, beta in version Y, stable in version Z. I am afraid people will try this now with 1.22 and get something different than described here.